File: output/StarCoder/codenet/C++/Java/s914373115.java
import java.util.*;
public class s914373115 {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        long x = in.nextLong();
        long sum = 100;
        int i = 0;
        while (sum < x) {
            sum += sum / 100;
            i++;
        }
        System.out.println(i);
    }
}

File: output/StarCoder/codenet/C++/Python/s914373115.py
def A():
    pass

def B():
    x = int(input())
    sum = 100
    i = 0
    while sum < x:
        sum += sum / 100
        i += 1
    print(i)

def C():
    pass

def D():
    pass

def main():
    B()

if __name__ == '__main__':
    main()